### PR TITLE
Add functionality to send login credentials via email

### PR DESCRIPTION
**Pull request by dpkrane**
In my organization, there was a need to add the capability for bulk sending of usernames and passwords to users. 
I’ve implemented the required functionality. 
Hopefully, this will be useful for others as well.

### CODE REVIEW

File: app/operators/library/ajax/user_actions.php [hunk 1]
```diff
@@ -164,22 +157,67 @@
...
+            // Iterate through the results
+            while ($row = $res->fetchRow()) {
+                // Get the recipient's email address and username
+                $recipient_email_address = $row[2]; // Email of the user
+                $recipient_name = $row[0]; // Username of the user
+
+                // Set the subject and body of the email
+                $subject = 'VPN Credentials'; // Subject of the email
+                $body = sprintf(
+                    '<b>VPN credential</b><br>Hello, %s %s!<br>Your login is: %s<br>Your password is: %s<br>VPN Server is: vpn.company.com<br><br>Best regards, Admin',
```
Comment (lirantal) [inline app/operators/library/ajax/user_actions.php:190]: what is the `vpn.company.com` for?

Could you make that variable configurable from the $configValues['...'] and put a placeholder in the config files so others can easily change it?

**Review by lirantal — CHANGES_REQUESTED** [review]

File: app/operators/library/ajax/user_actions.php (hunk 1 above)
Comment (dpkrane) [reply app/operators/library/ajax/user_actions.php]: Done.

**Comment by dpkrane** [comment]
> Looks good to me @dpkrane, thanks for sending this over. What do you think about the comment I've made? I'd be happy to merge this in if you can address that bit.
> 
> p.s. would be cool if you can also add some screenshots of the screens :-)
screenshot2
screenshot1

**Comment by lirantal** [comment]
Looks awesome. Thank you!

**Review by lirantal — APPROVED** [review]

**Pull request merged**